vespa-osgi-testrunner/src/test/java/com/yahoo/vespa/testrunner/HtmlLoggerTest.java

```diff
@@ -0,0 +1,32 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.testrunner;
+
+import org.fusesource.jansi.Ansi;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.logging.LogRecord;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author jonmv
+ */
+class HtmlLoggerTest {
+
+    @Test
+    void testConversionToHtml() {
+        String splitMessage = Ansi.ansi().fg(Ansi.Color.RED).a("</body>Hello!\ncontinued").reset().toString();
```

Comment:
What will 2 spaces be represented as?

Reply:
2 space.

Reply:
so that will be collapsed to 1 space by HTML rendering then?

Reply:
Not any longer!

Reply:
Was able to fix that in the console code, using `pre-wrap'.